docs/contributor/03-10-hyperscaler-account-pool.md

```diff
@@ -82,3 +83,155 @@ metadata:
+
+```
+hap: 
+  platformRegionRule: "gcp:cf-sa30;gcp:cf-jp30;aws:cf-eu11;openstack"
+  clusterRegionRule: ""
+  sharedRule: ""
+  euAccessRule: ""
+```
+
+![hap-workflow](../assets/gcp-platform-region-pool-list.drawio.svg)
+
+---
+
+Below configuration specifies azure plan as one for which to use cluster region based search. In this case, cluster provisioned in with the azure plan would require a secret binding with `hyperscalerType: azure_<CLUSTER_REGION>` label. All of the rule examples apply all of platformRegionRule, clusterRegionRule, shareRule and euAccessRule properties.
+
+```
+hap: 
+  platformRegionRule: "gcp"
+  clusterRegionRule: "azure"
+  sharedRule: ""
+  euAccessRule: ""
+```
+
+![hap-workflow](../assets/gcp-azure-pool.drawio.svg)
+
+---
+
+All the configuration rules are independent of each other from the point of view of their configuration. If the same plan/region pair appears in more than one property, then all the rules that it appears in take effect. Below configuration specifies that for all gcp clusters the search should be done based on platform region and cluster region searches. The resulting search label would be `hyperscalerType: gcp_<PLATFORM_REGION>_<CLUSTER_REGION>`.
+
+```
+hap: 
+  platformRegionRule: "gcp"
+  clusterRegionRule: "gcp"
+  sharedRule: ""
+  euAccessRule: ""
+```
+
+![hap-workflow](../assets/gcp-cluster-pool.drawio.svg)
+
+---
+
+`sharedRule` and `euAccessRule` are idependent but in contrast to `platformRegionRule` and `clusterRegionRule` if they are both evaluated to true the search label contains two additional labels `shared: true` and `euAccess: true`. Instead of modifying the same label value `hyperscalerType` two additional labels are added to the same query. In the example below, all gcp clusters will use the same pool of secret bindings marked with labels: `hyperscalerType: gcp_<PLATFORM_REGION>_<CLUSTER_REGION>`, `shared: true`, `euAccess: true`.
+
+```
+hap: 
+  platformRegionRule: "gcp"
+  clusterRegionRule: "gcp"
+  sharedRule: "gcp"
+  euAccessRule: "gcp"
+```
+
+![hap-workflow](../assets/all-gcp-pool.drawio.svg)
+
+---
+
+The last example shows initial configuration create to mimic the current bahaviour of KEB at the time of writing the document. The configuration enforces that:
+* azure, aws, gcp have their own pools of dedicated bindings.
+* gcp clusters in the region cf-sa30 use the pool of secret bindings marked with labels: `hyperscalerType: gcp_cf-sa30`,
+* sap-converged-cloud clusters use the pool of secret bindings marked with labels: `hyperscalerType: openstack_<CLUSTER_REGION>` and all of the are shared.
+* trial clusters can use one of two pool of shared secret bindings marked with labels: `hyperscalerType: azure` or `hyperscalerType: aws` depending on the provider type.
```

Comment:
It is worthwhile to mention that that trial and provider type mappings are hardcoded in the code. To make it clear, the trial plan is treated differently than other plans.

Reply:
It is hard for me to find a place in that document for that remark. Modification of the commented line might obscure its relation the the configuration example. Other places in the document are not suitable as well because this is not a document about plans but about HAP. However, I have added a note about this in https://github.com/kyma-project/kyma-environment-broker/pull/1643/commits/6bab7851882a8e0cff3f8170e1168362d8ec860c